scribus/langmgr.cpp

```diff
@@ -89,90 +89,541 @@ void LanguageManager::generateLangList()
+	m_langTable.append(LangDef("dz",       "",         "Dzongkha",                            QObject::tr("Dzongkha")));
+	m_langTable.append(LangDef("igb",      "",         "Ebira",                               QObject::tr("Ebira")));
+	m_langTable.append(LangDef("efi",      "",         "Efik",                                QObject::tr("English (Efik)")));
+	m_langTable.append(LangDef("en_AU",    "",         "English (Australia)",                 QObject::tr("English (Australia)")));
+	m_langTable.append(LangDef("en_CA",    "",         "English (Canada)",                    QObject::tr("English (Canada)")));
+	m_langTable.append(LangDef("en_NZ",    "",         "English (New Zealand)",               QObject::tr("English (New Zealand)")));
+	m_langTable.append(LangDef("en_ZA",    "",         "English (South Africa)",              QObject::tr("English (South Africa)")));
+	m_langTable.append(LangDef("en_GB",    "en",       "English (UK)",                        QObject::tr("English (UK)")));
+	m_langTable.append(LangDef("en_US",    "",         "English (US)",                        QObject::tr("English (US)")));
+	m_langTable.append(LangDef("ang",      "",         "English (Old ca. 450-1100)",          QObject::tr("English (Old ca. 450-1100)")));
+	m_langTable.append(LangDef("eo",       "",         "Esperanto",                           QObject::tr("Esperanto")));
+	m_langTable.append(LangDef("myv",      "",         "Erzya",                               QObject::tr("Erzya")));
+	m_langTable.append(LangDef("et",       "et_EE",    "Estonian",                            QObject::tr("Estonian")));
+	m_langTable.append(LangDef("ekk",      "",         "Estonian (Standard)",                 QObject::tr("Estonian (Standard)")));
+	m_langTable.append(LangDef("eve",      "",         "Even",                                QObject::tr("Even")));
+	m_langTable.append(LangDef("evn",      "",         "Evenki",                              QObject::tr("Evenki")));
+	m_langTable.append(LangDef("ee",       "",         "Ewe",                                 QObject::tr("Ewe")));
+	m_langTable.append(LangDef("fo",       "",         "Faroese",                             QObject::tr("Faroese")));
+	m_langTable.append(LangDef("fj",       "",         "Fijian",                              QObject::tr("Fijian")));
+	m_langTable.append(LangDef("fil",      "",         "Filipino",                            QObject::tr("Filipino")));
+	m_langTable.append(LangDef("fi",       "",         "Finnish",                             QObject::tr("Finnish")));
+	m_langTable.append(LangDef("fon",      "",         "Fon",                                 QObject::tr("Fon")));
+	m_langTable.append(LangDef("myq",      "",         "Forest Maninka",                      QObject::tr("Forest Maninka")));
+	m_langTable.append(LangDef("frp",      "",         "Franco-Provençal",                     QObject::tr("Franco-Provençal")));
```

Comment:
I think `QObject::tr()` does not work well with non-ASCII characters, you need to escape them (see Bokmål below).